modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -104,7 +105,9 @@ export default {
       next: klona(this.items),
       triggerValidation: false,
       minError: false,
-      maxError: false
+      maxError: false,
+      editing: false,
```

Comment:
We don't seem to be using `editing` anymore.

Reply:
We should remove that here.

Reply:
Oops pushed that fix up for real this time.